store/postgres/src/relational.rs

```diff
@@ -539,6 +539,32 @@ impl Layout {
         Ok(entities)
     }
 
+    pub fn find_derived(
+        &self,
+        conn: &PgConnection,
+        derived_query: &DerivedEntityQuery,
+        block: BlockNumber,
+        excluded_keys: &Option<Vec<EntityKey>>,
```

Comment:
Hahaha, that's funny! I was doing exactly this and changed to `Option` because I saw in some internet post that we should use Option on  "null values" like `-1`, empty string `""` and I thought a `vec![]` would match this case.

But indeed, in this case there is no difference between `None` and `Some(vec![])`. I was just doing two checks instead of one.